### DISCUSSION TITLE
How can I convert spectral distributions to dictionary structure?

### DISCUSSION THREAD

**Discussion by barisduzenli** [colour-science/colour#1100] [Q&A] [answered]
Hello,

I am trying to convert spectral distributions to dictionary structure. I simply tried

SpectralDistribution_Dict = dict(map(reversed, SpectralDistribution))

and I got this error

AttributeError: "SpectralDistribution.__iter__" object has been removed from the API.

**Comment by thomasmansencal** [comment, marked as answer]
Hi @barisduzenli,

It should be as simple as that:

```python
import colour

sd = colour.SDS_ILLUMINANTS["D65"]

print(dict(zip(sd.wavelengths, sd.values)))
```

Cheers,

Thomas

**Reply by barisduzenli** [reply]
Hello Thomas,

Yes, it is just like easy.

Thank you.
